### PR TITLE
006 job role responsibilities

### PR DESCRIPTION
**Pull request by matthewmcpolinkainos**

### CODE REVIEW

**Review by walkinator16000 — APPROVED** [review]
Looks great! 😀 👍

File: .idea/runConfigurations.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (felixmoore) [inline .idea/runConfigurations.xml:1]: Is this file necessary?

**Review by felixmoore — APPROVED** [review]

**Review by walkinator16000 — APPROVED** [review]
all good again!
